### PR TITLE
Added sleep before exit on failure and logged URLs of failing requests

### PR DESCRIPTION
**Pull request by davidbalazs93**
Changes in this PR:
- Added sleep before exit on failure 
- Logged URLs of failing requests

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,5 +14,5 @@ RUN apk --update add go git gcc linux-headers libc-dev \
   && apk del go git gcc linux-headers libc-dev \
   && rm -rf $GOPATH /var/cache/apk/*
```
Comment (tamas-molnar) [inline Dockerfile]: Consider installing curl or wget so we can debug faster in the container.

File: main.go [hunk 2]
```diff
@@ -75,11 +77,14 @@ func main() {
 	}
 
 	for {
-		log.Printf("Starting deploy run")
+		log.Print("Starting deploy run")
 		if err := d.deployAll(); err != nil {
-			log.Fatalf("Failed to run deploy : %v\n", err.Error())
+			log.Printf("Failed to run deploy : %v\n", err.Error())
+			time.Sleep(time.Duration(*noOfSecsToSleepBeforeRestart) * time.Second)
```
Comment (tamas-molnar) [inline main.go]: Can you add a log before sleeping, so we know what the app is doing?

File: Dockerfile (hunk 1 above)
Comment (khatton-ft) [inline Dockerfile]: wget is already present in the container.

**Comment by khatton-ft** [comment]
Looks good, just need to add a log saying that the app is sleeping as per @tamas-molnar's suggestion.

File: Dockerfile (hunk 1 above)
Comment (davidbalazs93) [inline Dockerfile]: yes, wget is already present in the container

File: main.go (hunk 2 above)
Comment (davidbalazs93) [inline main.go]: yes, I'm adding it now

**Pull request merged**